.github/workflows/wasm_editor_and_interpreter.yaml

```diff
@@ -14,41 +15,109 @@ jobs:
+
+      - name: Build slint-wasm-interpreter
+        run: npm run build:wasm_preview-release
+        working-directory: tools/online_editor
+      - name: Build slint-wasm-lsp
+        run: npm run build:wasm_lsp-release
+        working-directory: tools/online_editor
+
+      - name: "Upload wasm Artifacts"
+        uses: actions/upload-artifact@v2
+        with:
           name: wasm
           path: |
-              api/wasm-interpreter/pkg/
-              tools/lsp/pkg/
-              tools/online_editor/dist/
+            api/wasm-interpreter/pkg/
+            tools/lsp/pkg/
+
+  build-online-editor:
+    needs: [wasm]
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/install-nodejs
+      - name: Download the WASM binaries
+        uses: actions/download-artifact@v3
+        with:
+          name: wasm
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-${{ github.job }}-node-${{ hashFiles('api/wasm-interpreter/pkg/package.json', 'tools/lsp/pkg/package.json', 'tools/online_editor/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ github.job }}-node-
+
+      - name: Install NPM dependencies
+        run: npm install
+        working-directory: tools/online_editor
+
+      - name: Compile online editor
+        run: npm run build
+        working-directory: tools/online_editor
+
+      - name: "Upload online_editor Artifacts"
+        uses: actions/upload-artifact@v2
+        with:
+          name: online_editor
+          path: tools/online_editor/dist/
+
+  lint-online-editor:
+    needs: [wasm]
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/install-nodejs
+      - name: Download the WASM binaries
+        uses: actions/download-artifact@v3
+        with:
+          name: wasm
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-${{ github.job }}-node-${{ hashFiles('api/wasm-interpreter/pkg/package.json', 'tools/lsp/pkg/package.json', 'tools/online_editor/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ github.job }}-node-
+
+      - name: Install NPM dependencies
+        run: npm install
+        working-directory: tools/online_editor
+
+      - name: Lint online editor  # This needs the slint-wasm-interpreter!
+        run: |
+          npm run syntax_check
+          npm run lint
+        working-directory: tools/online_editor
+
+  test-online-editor-ff:
+    needs: [wasm]
+    runs-on: ubuntu-latest
+    container:
+      image: cypress/browsers:node16.13.0-chrome95-ff94
+      options: --user 1001
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/test-online-editor
+        with:
+          browser: "firefox"
+
+  test-online-editor-chrome:
+    needs: [wasm]
```

Comment:
I think you somehow need to copy the artifact over

Reply:
It's fine for this simple smoke test, but yes, I should do that :-)